conftest.py

```diff
@@ -50,7 +52,8 @@ def pytest_addoption(parser):
         "--test-databases",
         nargs="+",
         default=['mongo'],
-        help="Test databases in the list. Note that travis-ci will test all of them by default.")
+        help=("Test databases in the list. List items can include: " + db_names +
```

Comment:
Sorry, I apparently couldn't resist adding an improvement to the help text while I was working in this file.